NorthstarDLL/server/serverchathooks.h

```diff
@@ -9,7 +9,7 @@ enum class CustomMessageType : char
 };
 
 constexpr unsigned char CUSTOM_MESSAGE_INDEX_BIT = 0b10000000;
-constexpr unsigned char CUSTOM_MESSAGE_INDEX_MASK = ~CUSTOM_MESSAGE_INDEX_BIT;
+constexpr unsigned char CUSTOM_MESSAGE_INDEX_MASK = (unsigned char)~CUSTOM_MESSAGE_INDEX_BIT;
```

Comment:
bitwise operations implicitly turn into an integers so put it back into its place.